src/javascript/app/pages/trade/digit_trade.js

```diff
@@ -0,0 +1,94 @@
+const moment               = require('moment');
+const DigitTicker          = require('./digit_ticker');
+const ViewPopupUI          = require('../user/view_popup/view_popup.ui');
+const showLocalTimeOnHover = require('../../base/clock').showLocalTimeOnHover;
+const BinarySocket         = require('../../base/socket');
+const localize             = require('../../../_common/localize').localize;
+const getPropertyValue     = require('../../../_common/utility').getPropertyValue;
+
+const DigitDisplay = (() => {
+    let $container,
+        contract,
+        tick_count;
+
+    const init = (id_render, proposal_open_contract) => {
+        tick_count = 1;
+        contract   = proposal_open_contract;
+
+        $container = $(`#${id_render}`);
+        $container
+            .addClass('normal-font')
+            .html($('<h5 />', { text: contract.display_name, class: 'center-text' }))
+            .append($('<div />', { class: 'gr-6 gr-centered' })
+                .append($('<div />', { class: 'gr-row', id: 'table_digits' })
+                    .append($('<strong />', { class: 'gr-3', text: localize('Tick') }))
+                    .append($('<strong />', { class: 'gr-3', text: localize('Spot') }))
+                    .append($('<strong />', { class: 'gr-6', text: localize('Spot Time (GMT)') }))))
+            .append($('<div />', { class: 'digit-ticker invisible', id: 'digit-ticker-container' }));
+
+        const request     = {
+            ticks_history: contract.underlying,
+            start        : contract.date_start,
+        };
+        if (contract.current_spot_time < contract.date_expiry) {
```

Comment:
perhaps a comment above this line to explain this means that the contract is still ongoing/streaming